projects/packages/external-media/src/features/admin/external-media-import.js

```diff
@@ -24,20 +24,33 @@ const JetpackExternalMediaImport = () => {
 		}
 	};
 
+	const selectButtonText = ( selectedImages, isCopying ) => {
+		if ( isCopying ) {
+			return sprintf(
+				/* translators: %1$d is the number of media that were selected. */
+				__( 'Importing… %1$d media', 'jetpack-external-media' ),
+				selectedImages
+			);
+		}
+
+		return selectedImages
+			? sprintf(
+					/* translators: %1$d is the number of media that were selected. */
+					__( 'Import %1$d media', 'jetpack-external-media' ),
+					selectedImages
+			  )
+			: __( 'Import media', 'jetpack-external-media' );
+	};
+
 	const handleSelect = media => {
 		if ( ! media || media.length === 0 ) {
 			return;
 		}
 
 		showNotice(
 			sprintf(
-				/* translators: %d is the number of the media file */
-				_n(
-					'%d media file imported successfully',
-					'%d media files imported successfully',
-					media.length,
-					'jetpack-external-media'
-				),
+				/* translators: %d is the number of the media */
```

Comment:
Is there a specific reason why we changed the term `media file` to `media` everywhere in the PR?

Reply:
The copy on the button of the modal is `Importing %d media`, so it seems to be better to get rid of the `file` IMO. What do you think?